src/panels/flow-histogram-react/FlowHistogramHelpers.ts

```diff
@@ -215,3 +216,67 @@ export const getSeriesMetricValues = (fields: any[], name: string) => {
 
 }
 
+export const setLegend = (options: { flowHistogramOptions: FlowHistogramOptionsProps }) => {
+    if (options.flowHistogramOptions.showLegend) {
+        const className = options.flowHistogramOptions.position.label === 'Under Graph' ? '.flow-histogram-legend-bottom' : '.flow-histogram-legend-right'
```

Comment:
I know this is a workaround for the legend issue with Flot, but make a note for yourself to remove this in a later update. This jquery/html solution looks out of place in the context of a React plugin.